Source/MonoGame.Extended/Particles/Profiles/Profile.cs

```diff
@@ -1,60 +1,48 @@
 ﻿using Microsoft.Xna.Framework;
+using MonoGame.Extended.Shapes;
 
 namespace MonoGame.Extended.Particles.Profiles
 {
     public abstract class Profile //: ICloneable
     {
-        public abstract void GetOffsetAndHeading(out Vector2 offset, out Axis heading);
+        public abstract void GetOffsetAndHeading(out Vector2 offset, out Vector2 heading);
 
-        public object Clone()
-        {
+        public object Clone() {
             return MemberwiseClone();
         }
 
         public enum CircleRadiation { None, In, Out }
 
-        public static Profile Point()
-        {
-            return new PointProfile();
+        public static Profile Point(PointF point) {
+            return new OutlineProfile(point);
         }
 
-        public static Profile Line(Axis axis, float length)
-        {
-            return new LineProfile { Axis = axis, Length = length };
+        public static Profile Line(LineF line) {
+            return new OutlineProfile(line);
         }
 
-        public static Profile Ring(float radius, CircleRadiation radiate)
-        {
-            return new RingProfile { Radius = radius, Radiate = radiate };
+        public static Profile Ring(CircleF circle, CircleRadiation radiate) {
+            return new OutlineProfile(circle, radiate, circle.Center);
```

Comment:
Why is constructor injection better here? In most cases I would say constructor injection is a good thing but at some point we are going to want to deserailize these objects from a file and constructors can get in the way.